### ISSUE TITLE
Should disabling gzip require the webpack plugin?

### ISSUE THREAD

**Issue by ziir** [ai/size-limit#163]
Hello,

Thanks for this very interesting package, I'm testing it on a pet project app to evaluate using it on work projects (apps, big and small libraries).

It works fine with the default options:

```json
  "size-limit": [
    {
      "path": "dist/js/app.*.js"
    }
  ],
```
```
  Size:         31.35 KB
  Loading time: 627 ms   on slow 3G
  Running time: 207 ms   on Snapdragon 410
  Total time:   834 ms
```

It appears that in current version (`size-limit` and `@size-limit/preset-app` 4.0.1), disabling gzip via `gzip: false` requires the webpack plugin:

```diff
  "size-limit": [
    {
-      "path": "dist/js/app.*.js"
+      "path": "dist/js/app.*.js",
+      "gzip": false
    }
  ],
```
```
$ npx size-limit
ERROR  Config option gzip needs @size-limit/webpack plugin
```

I've quickly patched `size-limit/size-limit-error.js`:

```diff
-   gzip: 'webpack',
+   gzip: true,
```

```
$ npx size-limit
Size:         101.1 KB
Loading time: 2.1 s    on slow 3G
Running time: 164 ms   on Snapdragon 410
Total time:   2.2 s
```

Which is now closer to the desired output (printing the uncompressed size).

Is the webpack plugin being required for `gzip: false` expected?
If so, should the documentation mention that?

Thank you for any response,

**Comment by ai** [comment]
Thanks for the report.

Fixed fe3f644

**Comment by ai** [comment]
The fix (now we `gzip` works with `webpack` **or** `file` plugin) was released in 4.0.2
